src/analysis/special_functions/log/deriv.lean

```diff
@@ -268,4 +268,57 @@ begin
       by simpa [pow_succ'] using mul_le_of_le_one_right (pow_nonneg (abs_nonneg x) i) (le_of_lt h) }
 end
 
+/-- Power series expansion of `log(1 + x) - log(1 - x)` for `|x| < 1`. -/
+lemma has_sum_log_sub_log_of_abs_lt_1 {x : ℝ} (h : |x| < 1) :
+  has_sum (λ k : ℕ, (2 : ℝ) * (1 / (2 * k + 1)) * x ^ (2 * k + 1)) (log (1 + x) - log(1 - x)) :=
+begin
+  have h₁ := has_sum_pow_div_log_of_abs_lt_1 h,
+  have h₂ := has_sum.mul_left (-1) (has_sum_pow_div_log_of_abs_lt_1 (eq.trans_lt (abs_neg x) h)),
+  rw [neg_one_mul, neg_neg, sub_neg_eq_add 1 x] at h₂,
```

Comment:
These are used only inside the goal that starts at line 286, so I would move them inside those braces.

Reply:
Done. Also moved `h₁` and `h₂`,  inlined `h₁`, renamed  `h₂` to `h₁`.